.gitignore

```diff
@@ -7,4 +7,6 @@ MANIFEST
 .cache/
 *.egg-info/
 htmlcov/
-.pytest_cache
\ No newline at end of file
+.pytest_cache
+**/.vscode
```

Comment:
Please remove those changes. I don't want to maintain an growing list of ignored files and directories that IDEs, editors and the next wow-so-fancy-tool are adding.
IMO those ignores belong in your global gitignore file since it's your personal preference to work with VS Code etc.

Reply:
Done.